app/controllers/api/v1x0/portfolios_controller.rb

```diff
@@ -9,9 +9,10 @@ def index
       end
 
       def create
-        authorize(Portfolio)
+        portfolio = authorize(Portfolio.new(writeable_params_for_create))
 
-        portfolio = Portfolio.create!(writeable_params_for_create)
+        portfolio.save!
+        portfolio.reload
```

Comment:
Is this line necessary? I know we would have to do something like this for specs, but I'm pretty sure we don't have to worry about it in the controller.

Reply:
@eclarizio I have removed it, I wasn't sure how the ID would get updated in the new object after the save! without a reload